.github/workflows/supported_storages_compatibility_check.yaml

```diff
@@ -0,0 +1,728 @@
+          cp -a core/build/reports/tests/test /tmp/gradle_test_reports/core/
+          cp -a server/build/reports/tests/test /tmp/gradle_test_reports/server/
+          cp -a schema-loader/build/reports/tests/test /tmp/gradle_test_reports/schema-loader/
+
+      - name: Upload Gradle test reports
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: gradle_check_reports
+          path: /tmp/gradle_test_reports
+
+      - name: Save SpotBugs reports
+        if: always()
+        run: |
+          mkdir -p /tmp/gradle_spotbugs_reports/core
+          mkdir -p /tmp/gradle_spotbugs_reports/server
+          mkdir -p /tmp/gradle_spotbugs_reports/schema-loader
+          mkdir -p /tmp/gradle_spotbugs_reports/integration-test
+          cp -a core/build/reports/spotbugs /tmp/gradle_spotbugs_reports/core/
+          cp -a server/build/reports/spotbugs /tmp/gradle_spotbugs_reports/server/
+          cp -a schema-loader/build/reports/spotbugs /tmp/gradle_spotbugs_reports/schema-loader/
+          cp -a integration-test/build/reports/spotbugs /tmp/gradle_spotbugs_reports/integration-test/
+
+      - name: Upload Spotbugs reports
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: spotbugs_reports
+          path: /tmp/gradle_spotbugs_reports
+
+  dockerfile-lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+
+      - name: Dockerfile Lint for ScalarDB Server
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: ':server:dockerfileLint'
+
+      - name: Dockerfile Lint for ScalarDB Schema Loader
+        if: always()
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: ':schema-loader:dockerfileLint'
+
+  cassandra-2-2-integration-test:
+    runs-on: ubuntu-latest
+
+    services:
+      cassandra:
+        image: cassandra:2.2.19
+        env:
+          MAX_HEAP_SIZE: 2048m
+          HEAP_NEWSIZE: 512m
+        ports:
+          - 9042:9042
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+
+      - name: Setup and execute Gradle 'integrationTestCassandra' task
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: integrationTestCassandra
+
+      - name: Upload Gradle test reports
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: cassandra_2.2_integration_test_reports
+          path: core/build/reports/tests/integrationTestCassandra
+
+  cassandra-3-0-integration-test:
+    runs-on: ubuntu-latest
+
+    services:
+      cassandra:
+        image: cassandra:3.0.29
```

Comment:
How about specifying just `cassandra:3.0` so that we don't need to update the patch version frequently?

Reply:
That is a good point, thank you. I will update it.

Reply:
I updated all the images in the workflow to use the latest major version besides Cassandra and Mysql, which use the latest minor version.